### PR TITLE
docs(tootltip): update docker trigger tooltip

### PR DESCRIPTION
**Pull request by emjburns**
Adding guidance around `latest` tag and tag updates.

### CODE REVIEW

File: app/scripts/modules/core/src/help/help.contents.ts [hunk 1]
```diff
@@ -282,7 +282,7 @@ const helpContents: { [key: string]: string } = {
   'pipeline.config.script.propertyFile':
     '<p>(Optional) The name to the properties file produced by the script execution to be used by later stages of the Spinnaker pipeline. </p>',
   'pipeline.config.docker.trigger.tag':
-    '<p>(Optional) If specified, only the tags that match this Java Regular Expression will be triggered. Leave empty to trigger builds on any tag pushed.</p>',
+    '<p>(Optional) If specified, only the tags that match this Java Regular Expression will be triggered. Leave empty to trigger builds on any tag pushed.</p><p>Builds will not be triggered off the latest tag or tag updates.</p>',
```
Comment (lwander) [inline app/scripts/modules/core/src/help/help.contents.ts]: Can you rephrase this to "updates to existing tags" instead of "tag updates"?

Comment (emjburns) [reply app/scripts/modules/core/src/help/help.contents.ts]: Done.
